wazo_acceptance/helpers/sip_phone.py

```diff
@@ -127,7 +120,17 @@ def is_hungup(self):
         return self._session.hook_status() == HookStatus.OFFHOOK
 
 
-def register_line(sip_config):
-    phone = SipPhone(sip_config)
-    phone.register()
-    return phone
+class RegisterLine:
```

Comment:
I'm not too found the the `RegisterLine` name especially since the class has a `register_line` method.

How about `LineRegistrar`?

Reply:
Haha yeah, I wasn't inspired

Reply:
fixed